Remove last bits of Backbone, Marionette, and jQuery

maxlath opened the pull request:
Internalizes minimal implementations of a router, and event bus, mimicking respectively `Backbone.Router` and `backbone.radio`.

`Backbone.history` is replaced by the native [History API](https://developer.mozilla.org/en-US/docs/Web/API/History_API)

~~Depends on #538~~

jum-s commented on app/init_app_layout.ts:
```diff
@@ -0,0 +1,36 @@
+import { commands } from './radio'
+import type { SvelteComponent, ComponentProps, ComponentType } from 'svelte'
+
+export let appLayout
```
Loosening dependencies, love it!

jum-s commented:
when I click on "Add edition without ISBN", an error is shown: 

```
Error: TypeError: expected string, got undefined (undefined)
    newError webpack-internal:///./app/lib/error.ts:20
    assertType webpack-internal:///./app/lib/assert_types.ts:26
    assertString webpack-internal:///./app/lib/assert_types.ts:70
    showEntityCreateFromRoute webpack-internal:///./app/modules/entities/entities.ts:135
    loadUrl webpack-internal:///./app/lib/router.ts:51
    navigate webpack-internal:///./app/app.ts:72
    navigateAndLoad webpack-internal:///./app/app.ts:85
    loadInternalLink webpack-internal:///./app/lib/utils.ts:129
```

maxlath commented:
> when I click on "Add edition without ISBN", an error is shown

Fixed by f2a5664

maxlath commented:
> When I am on a list with element, I click on an element, then it glimpses and reload the whole list's page to open element modal. This behaviour is not happening on main.

Fixed on main by 3af75aeb5 (has it was caused by de69eaa3, which is already on main)

>  if I hit previous to go back to an element URL (lists/:id/element/:elId), nothing happen

Fixed by 5eb8db279

maxlath commented:
Merging as all reported problems have been fixed and this PR fixes known bugs currently in prod. But of course, if more issues are identified, they can be reported and addressed here.
